File: solutions/go/pangram/1/pangram.go
// This package determines if a sentence is a pangram
package pangram
import "strings"

// This function determines if a sentence is a pangram
func IsPangram(input string) bool {
	var amap = make(map[string]int)
	alphabet := "abcdefghijklmnopqrstuvwxyz"
	for _,l := range alphabet {
		amap[string(l)]=0
	}
	for _,l := range input {
		amap[strings.ToLower(string(l))] = 1
	}
	for _,v := range amap{
		if v == 0 {
			return false
		}
	}
	return true
}

File: solutions/csharp/pangram/1/Pangram.cs
using System;

public static class Pangram
{
    public static bool IsPangram(string input)
    {
        int[] charcount = new int[26];
        input = input.ToLower();
        foreach  (char ch in input) {
            if (ch>='a' && ch <='z') {
                charcount[ch-'a']++;
           }
        }

        for (int i = 0; i<26; i++) {
            if (charcount[i] == 0) return false;
        }
        return true;
    }
}
